src/main/scala/nl.knaw.dans.easy.deposit/DepositDir.scala

```diff
@@ -176,10 +176,11 @@ object DepositDir {
     if (userDir.exists)
       userDir
         .list
-        .filter(_.isDirectory)
+        .withFilter(_.isDirectory)
         .map(deposit => Try {
           DepositDir(draftDir, user, UUID.fromString(deposit.name))
         }.recoverWith { case t: Throwable => Failure(CorruptDepositException(user, deposit.name, t)) })
+        .withFilter(_.isSuccess)// CorruptDepositException logged itself
```

Comment:
What other type of Failures may result from this? You are doing a `collectResults` at the end, so I gather there may still be Failures in the sequence (otherwise the `collectResults` would not really make sense).

Reply:
Come to think of it, you are filtering all types of Failures, so the `collectResults` will never collect any Failures.